modules/ROOT/pages/authentication-authorization/manage-users.adoc

```diff
@@ -0,0 +1,860 @@
+The `SET PASSWORD` part is only optional if it directly follows the `SET PASSWORD` clause.
+* For `SET PASSWORD CHANGE [NOT] REQUIRED`, the `SET PASSWORD` is only optional if it directly follows the `SET PASSWORD` clause.
+* `SET HOME DATABASE` can be used to configure a home database for a user.
+A home database will be resolved if it is either pointing to a database or a database alias.
+If no home database is set, the DBMS default database is used as the home database for the user.
+* `REMOVE HOME DATABASE` is used to unset the home database for a user.
+This results in the DBMS default database being used as the home database for the user.
+
+For example, you can modify the user `bob` with a new password and active status, and remove the requirement to change his password:
+
+[source, cypher, role=noplay]
+----
+ALTER USER bob
+SET PASSWORD 'abcd5678' CHANGE NOT REQUIRED
+SET STATUS ACTIVE
+----
+
+Or you may decide to assign the user `bob` a different home database:
+
+[source, cypher, role=noplay]
+----
+ALTER USER bob
+SET HOME DATABASE anotherDbOrAlias
+----
+
+Or remove the home database from the user `bob`:
+
+[source, cypher, role=noplay]
+----
+ALTER USER bob
+REMOVE HOME DATABASE
+----
+
+[NOTE]
+====
+When altering a user, it is only necessary to specify the changes required.
+For example, leaving out the `CHANGE [NOT] REQUIRED` part of the query will leave that unchanged.
+====
+
+[NOTE]
+====
+The `SET STATUS {ACTIVE | SUSPENDED}`, `SET HOME DATABASE`, and `REMOVE HOME DATABASE` parts of the command are only available in Neo4j Enterprise Edition. label:enterprise-edition[]
+====
+
+The changes to the user will appear on the list provided by `SHOW USERS`:
+
+[source, cypher, role=noplay]
+----
+SHOW USERS
+----
+
+.Result
+[options="header,footer", width="100%", cols="2m,3m,3m,2m,2m"]
+|===
+|user |roles |passwordChangeRequired |suspended |home
+
+|"bob"
+|["PUBLIC"]
+|false
+|false
+|<null>
+
+|"neo4j"
+|["admin","PUBLIC"]
+|true
+|false
+|<null>
+
+5+a|Rows: 2
+
+|===
+
+The default behavior of this command is to throw an exception if the user does not exist.
+Adding an optional parameter `IF EXISTS` to the command makes it idempotent and ensures that no exception is thrown.
+Nothing happens should the user not exist.
+
+[source, cypher, role=noplay]
+----
+ALTER USER nonExistingUser IF EXISTS SET PASSWORD 'abcd1234'
+----
+
+
+[[access-control-alter-password]]
+== Changing the current user's password
+
+Users can change their password using `ALTER CURRENT USER SET PASSWORD`.
+The old password is required in addition to the new one, and either or both can be a string value or a string parameter.
+When a user executes this command it will change their password as well as set the `CHANGE NOT REQUIRED` flag.
+
+// can't test, don't want to hardcode test user password
+[source, cypher, role=test-skip]
+----
+ALTER CURRENT USER
+SET PASSWORD FROM 'password1' TO 'password2'
+----
+
```

Comment:
I'll assume it will be updated once additional PRs are ported afterwards but will leave the comment open to remember to check that if that's ok with you?